static/js/types/websites.ts

```diff
@@ -8,6 +8,10 @@ export enum WidgetVariant {
   String = "string",
   Select = "select"
 }
+export interface FieldCondition {
```

Comment:
is `FieldCondition` the right name for this? I'm not sure if there's been discussion about making it more general, but as I'm reading it now it looks to me like this limited to checking that a field equals a given value. Maybe the name should be `FieldValueCondition` ? I'm not sure, what do you think?

Reply:
I'll rename it to `FieldValueCondition` I don't know what the longer term plans are for conditional fields, if it's just checking for equality or if there's other stuff down the line. I imagine it's probably for select fields, something like if country == "US" then show the state dropdown, in which case I doubt it will get more complicated.